shared/src/zeroex_api.rs

```diff
@@ -224,69 +231,40 @@ mod tests {
     #[tokio::test]
     #[ignore]
     async fn test_api_e2e() {
-        let zeroex_client = DefaultZeroExApi::with_default_url(Client::new());
-        let sell_token = crate::addr!("EeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE");
-        let buy_token = crate::addr!("1a5f9352af8af974bfc03399e3767df6370d82e4");
+        let zeroex_client = DefaultZeroExApi::default();
         let swap_query = SwapQuery {
-            sell_token,
-            buy_token,
-            sell_amount: Some(1_000_000_000_000_000_000u128.into()),
+            sell_token: crate::addr!("c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"),
+            buy_token: crate::addr!("a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"),
+            sell_amount: Some(U256::from_f64_lossy(1e18)),
             buy_amount: None,
             slippage_percentage: Slippage(0.1_f64),
-            skip_validation: Some(true),
         };
 
         let price_response = zeroex_client.get_swap(swap_query).await;
+        dbg!(&price_response);
         assert!(price_response.is_ok());
     }
 
-    #[test]
-    fn swap_query_serialization_0x_sell_order() {
```

Comment:
The github diff around here isn't great. I removed the serialization tests because they would have to be kept up to date with any change to `format_url` above. That function is simple so I felt the tests didn't provide value.